aws/components/apigateway/state.ftl

```diff
@@ -83,6 +83,7 @@ created in either case.
 
     [#local certificatePresent = isPresent(solution.Certificate) ]
     [#local cfPresent          = isPresent(solution.CloudFront) ]
+    [#local cfPresent          = isPresent(solution.CloudFront) ]
```

Comment:
Looks redundant?

Reply:
Yep, Not sure how that was added in there. Removed now.